components/Hero.vue

```diff
@@ -1,31 +1,22 @@
 <script lang="ts" setup></script>
 
 <template>
-  <section class="flex-1 flex flex-col justify-center items-center px-6 py-12">
-    <div class="flex flex-col items-center w-full max-w-lg">
+  <section class="flex-1 flex flex-col items-center mt-6">
+    <div class="flex flex-col items-center max-w-3xl w-full">
```

Comment:
Remove `items-center` from the parent.
```suggestion
    <div class="flex flex-col items-center max-w-3xl w-full mx-auto">
```